platform/android/java_glue.cpp

```diff
@@ -55,6 +60,7 @@ struct jvalret {
 	jvalret() { obj = NULL; }
 };
 
+//@TODO evaluate,this function doesn't seem to be used anywhere.. Do we need it?
```

Comment:
The method is used in [JNISingleton::call](https://github.com/godotengine/godot/blob/40acc3ecdc4ca63040fec3ea30eead21523bade6/platform/android/java_glue.cpp#L462).

Reply:
Ah ok, removed the remark